src/main/kotlin/platform/bukkit/creator/bukkit-platforms.kt

```diff
@@ -75,6 +77,10 @@ class PaperPlatformStep(parent: BukkitPlatformStep) : AbstractBukkitPlatformStep
         super.setupUI(builder)
         with(builder) {
             row("Paper manifest:") {
```

Comment:
```suggestion
            row("Paper Manifest:") {
```

Reply:
Not my change in the PR, I don't think this is worth changing (at least on my side).

Reply:
Its for consistency. Everything else is uppercase. Only manifest is lowercase. And since you currently touch this area I figured you can push a little commit to change it.

Reply:
I don't have enough knowledge about the rest of the UI to tell you if this is actually convention or not so I don't feel comfortable with making the change. If we have a project maintainer to confirm this, that would be great. Otherwise I'll see if I find time to go through the rest of the UI to check.

Reply:
This is correct, the other labels of the creator use title case, so feel free to apply it if you want to. I can do it easily later anyway.

Reply:
Okay 👍 I'll switch it, thanks for the confirmation :)